event/views.py

```diff
@@ -1,9 +1,39 @@
+from django.core.exceptions import ObjectDoesNotExist
 from django.http import HttpResponseRedirect
 from django.shortcuts import render
-from creation.models import Event 
+from creation.models import Event, Group, User
+from Home.forms import Join
+
+def join(request):
+    if request.method == 'POST':
```

Comment:
The join view is already in the Home app so we don't need it here and dont worry about dividing the groups I believe Silvena is working on that.